content/en/docs/reference/config/security/authorization-policy/index.html

```diff
@@ -101,17 +101,14 @@
     - operation:
         methods: [&quot;POST&quot;]
         ports: [&quot;8080&quot;]
-
-When this rule is applied to TCP traffic, the `method` field (as will all HTTP based attributes) cannot be processed. 
-For a `DENY` rule, missing attributes are treated as matches. This means all TCP traffic on port 8080 would be denied in the example above.
-If we were to remove the `ports` match, all TCP traffic would be denied. As a result, it is recommended to always scope `DENY` policies to a specific port, 
-especially when using HTTP attributes [Authorization Policy for TCP Ports] (https://istio.io/latest/docs/tasks/security/authorization/authz-tcp/).
-
-The following authorization policy sets the `action` to &quot;AUDIT&quot;. It will audit any GET requests to the path with the
-prefix &quot;/user/profile&quot;.
-
-```yaml
-apiVersion: security.istio.io/v1beta1
+</code></pre>
+<p>When this rule is applied to TCP traffic, the <code>method</code> field (as will all HTTP based attributes) cannot be processed.
+For a <code>DENY</code> rule, missing attributes are treated as matches. This means all TCP traffic on port 8080 would be denied in the example above.
+If we were to remove the <code>ports</code> match, all TCP traffic would be denied. As a result, it is recommended to always scope <code>DENY</code> policies to a specific port,
+especially when using HTTP attributes [Authorization Policy for TCP Ports] (<a href="/docs/tasks/security/authorization/authz-tcp/)">https://istio.io/latest/docs/tasks/security/authorization/authz-tcp/)</a>.</p>
```

Comment:
We know this link is causing the issue. Need to look at the script to see about getting it updated correctly.